src/main/scala/sbtbuildinfo/BuildInfoPlugin.scala

```diff
@@ -18,6 +18,11 @@ object BuildInfoPlugin extends sbt.AutoPlugin {
     val BuildInfoType = sbtbuildinfo.BuildInfoType
     type BuildInfoType = sbtbuildinfo.BuildInfoType
     val addBuildInfoToConfig = buildInfoScopedSettings _
+
+    val buildInfoValues: TaskKey[Seq[BuildInfoResult]] =
+      taskKey("BuildInfo keys/values for use in the sbt build")
+    val buildInfoStringValues: TaskKey[Seq[(String,String)]] =
+      taskKey("BuildInfo keys/values for use in the sbt build as plain Strings")
```

Comment:
Anything that's not essential to generating build info Scala file should be left out of this plugin.